blog-homepage/src/Bloglist.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Bloglist.css';
+import Picture from './Picture'; 
+import AuthorThumbnail from './AuthorThumbnail';
+import missedArticles from './_data/missed-articles.json';
+import yourArticles from './_data/your-articles.json';
+import logo from './bookmark.png';
+import starIcon from './star.svg';
+import audioIcon from './audioIcon2.svg';
+import Moment from 'moment'; 
+import AuthorInfo from './AuthorInfo';
+
+
+class Bloglist extends React.Component{
+ 
+
+    static propTypes= {
+        src: PropTypes.string.isRequired
+    }
+
+    render(){
+        
+
+        return (
+           
+            <div className="Blog-ForYou">
```

Comment:
typically `classNames` are lower case and dash separated `blog-for-you`, same with ids